server/paths/api.ts

```diff
@@ -63,5 +64,8 @@ export default {
     prisonCaseNotes: personPath.path('prison-case-notes'),
     adjudications: personPath.path('adjudications'),
     offences: personPath.path('offences'),
+    oasys: {
```

Comment:
I think this should be namespaced under `people`

Reply:
It already is? or do you mean remove the `oasys` object and have it as something like `oasysSelection`?

Reply:
D'oh! Sorry!